src/utils/manageLocalStorage.ts

```diff
@@ -1,22 +1,20 @@
 import { GroupInfo } from "./globalTypes";
 
-const getLocalGroups = (): GroupInfo[] | [] => {
```

Comment:
This will always be an array of type `GroupInfo[]`, sometimes it'll just be an empty array, but it's legal to have an empty array with a type (`[] as GroupInfo[]`)